layouts/partials/ecosystem/registry/entry.html

```diff
@@ -167,7 +171,7 @@ <h6><i class="fa-solid fa-forward"></i> Quick Install</h6>
                 <div class="fw-bold">
                     {{ $languageNames.Get . | default (humanize .) }}
                 </div>
-                <small>Language</small>
+                <small data-bs-toggle="tooltip" title="This indicates the programming language used.">Language</small>
```

Comment:
It's not perfect, but we also have "Collector" as language. So add a sentence that "Collector" indicates that this is an extension for the OpenTelemetry Collector. If possible link to the collector documentation while doing so.